modules/rst.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+"""
+Add an |rst filter to the jinja templates…
+"""
+
+from docutils.core import publish_parts
+from jinja2 import Markup
+
+
+def rst_filter(s):
+    return Markup(publish_parts(source=s, writer_name='html')['body'])
```

Comment:
In `config.yml` we're using our own custom `html5writer`. Why the default `html` writer here?

rstblog also presets some other options for us, e.g. `initial_header_level`, `smart_quotes`, `syntax_highlight`.

Reply:
oh, great. I didn't know about those, but they'd be real nice to incorporate. Not sure I'm the one to do that…